static/sass/styles.scss

```diff
@@ -292,7 +292,6 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
   gap: 0;
 
   &__item {    
-    border-top: 1px solid $colors--light-theme--border-default;
```

Comment:
I would make the cursor a pointing hand when anywhere over the __item, as opposed to just the link inside